dev/io.openliberty.microprofile.telemetry.1.1.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -0,0 +1,330 @@
+    <modelVersion>4.0.0</modelVersion>
+    
+    <properties>
+        <arquillian.version>1.7.0.Alpha13</arquillian.version>
+		<arquillian.liberty.managed.jakarta.version>2.1.1</arquillian.liberty.managed.jakarta.version>
+		<testng.version>7.5.1</testng.version>
+		<slf4j.version>2.0.6</slf4j.version>
+
+        <opentelemetry.version>1.29.0</opentelemetry.version>
+        <mptelemetry.version>1.1</mptelemetry.version>
+
+        <!-- the below is used in arquillian.xml -->
+        <wlp>${wlp}</wlp>
+        <!-- <wlp>${project.basedir}/../../../../build.image/wlp/</wlp> -->
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <defaultSuiteFiles>tck-suite.xml</defaultSuiteFiles>
+        <suiteXmlFile>${defaultSuiteFiles}</suiteXmlFile>
+        <targetDirectory>${project.basedir}/target</targetDirectory>
+        
+    </properties>
+
+    <groupId>io.openliberty.microprofile.telemetry</groupId>
+    <artifactId>tck.runner.tck</artifactId>
+    <version>1.0</version>
+    <name>MicroProfile Telemetry ${mptelemetry.version} TCK Runner TCK Module</name>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>com.ibm.ws</groupId>
+            <artifactId>fattest.simplicity</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/../../../lib/fattest.simplicity.jar</systemPath> <!-- This will point to build/libs/autoFVT/lib/fattest.simplicity.jar -->
+        </dependency>
+        
+        <dependency>
+            <groupId>org.eclipse.microprofile.telemetry.tracing</groupId>
+            <artifactId>microprofile-telemetry-tracing-tck</artifactId>
+            <version>${mptelemetry.version}-RC2</version>
+        </dependency>
+        
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk</artifactId>
+            <version>${opentelemetry.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+            <version>${opentelemetry.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-instrumentation-annotations</artifactId>
+            <version>${opentelemetry.version}</version>
+        </dependency>
```

Comment:
The dependency list here is huge. It shouldn't need to be this long since the TCK should pull in all its dependencies automatically. Please raise an issue to investigate.

Reply:
Raised [#26085](https://github.com/OpenLiberty/open-liberty/issues/26085)
Opened in [OpenLiberty/open-liberty](https://github.com/OpenLiberty/open-liberty)